Wikipedia/Categories/UIColor+WMFStyle.m

```diff
@@ -54,6 +54,16 @@ + (instancetype)wmf_lightGrayColor {
     return c;
 }
 
++ (instancetype)wmf_placeholderLightGrayColor {
+    static UIColor* c = nil;
+
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        c = [UIColor colorWithWhite:0.975 alpha:1];
```

Comment:
yeah - it has to match the image that was given - so I used the value from that (this color is what makes the button the same gray as the placeholders fake lines of text)